### DISCUSSION TITLE
分辨率条件配置文件导致的卡顿

### DISCUSSION THREAD

**Discussion by 2607627233** [hooke007/mpv_PlayKit#172] [Q&A 问答] [answered]
1.控制台显示这个是不是缺少字幕相关字体
![QQ截图20220904200529](https://user-images.githubusercontent.com/43110396/188312642-bf063bcb-df69-41c3-8796-b4a8e5a26f7d.png)
2. 写了一个分辨率条件配置文件 ，但是每快进一次就加载一次。感觉加载都边卡顿了。有办法解决吗？
![QQ截图20220904200856](https://user-images.githubusercontent.com/43110396/188312699-c6114d8f-3d4d-4d58-953d-c340525cb919.png)

**Comment by hooke007** [comment, marked as answer]
1. 是
2. 展示你的内容，我不知道你写的是什么

**Reply by 2607627233** [reply]
[full-hd30] 
profile-desc=1080
profile-cond=((width ==1920 and height ==1080) and not p["video-frame-info/interlaced"] and p["estimated-vf-fps"]<31)
tscale=oversample

**Reply by hooke007** [reply]
https://github.com/hooke007/MPV_lazy/issues/154#issuecomment-1189657977
video-frame-info/interlaced 不可靠，不用它做自动条件配置的判断条件

跳转似乎会重置一部分属性导致重新触发配置开关，因此 estimated-vf-fps 中枪了
